experimental/Schemes/BlowupMorphism.jl

```diff
@@ -96,108 +97,193 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   return Z_trans, inc_Z_trans, pr_res
 end
 
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient wrapper to the MPolyIdeal-method
+function saturation_with_index(I::T,J::T) where T <: Union{ MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal}
+  R = base_ring(I)
+  R == base_ring(J) || error("Ideals do no live in the same ring.")
+
+  I_sat = saturated_ideal(I)
+  J_sat = saturated_ideal(J)
+
+  I_result,k = Oscar.saturation_with_index(I_sat,J_sat)  ## why is saturation_with_index not exported from Rings/mpoly-ideal.jl?
+  return (ideal(R,gens(I_result)),k)
+end
+
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient wrapper to the MPolyIdeal-method
+@attr Bool function is_one(I::Union{MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal})
+  return is_one(saturated_ideal(I))
+end
+
+@Markdown.doc """
+  strict_transform(p::BlowupMorphism, I::IdealSheaf)
+
+For a `BlowupMorphism`  ``p : Y → X`` and an `IdealSheaf` ``I`` on ``X`` return the
+strict transform of ``I`` on ``Y``.
+"""
 function strict_transform(p::BlowupMorphism, I::IdealSheaf)
+  Istrict,_ =_do_transform(p,I,-1)
+  return Istrict
+end
+
+@Markdown.doc """
+  weak_transform(p::BlowupMorphism, I::IdealSheaf)
+
+For a `BlowupMorphism`  ``p : Y → X`` and an `IdealSheaf` ``I`` on ``X`` return the
+weak transform of ``I`` on ``Y`` and the multiplicity of the exceptional divisor in the total transform of ``I``.
+"""
+function weak_transform(p::BlowupMorphism, I::IdealSheaf)
+  Iweak,_ =_do_transform(p,I,0)
+  return Iweak
+end
+
+function weak_transform_with_multiplicity(p::BlowupMorphism, I::IdealSheaf)
+  Iweak, multi = _do_transform(p,I,0)
+  return Iweak,multi
+end
+
+@Markdown.doc """
```

Comment:
```suggestion
@doc Markdown.doc"""
```
everywhere... Actually you might also be able to do 
```suggestion
@doc raw"""
```
or even
```suggestion
"""
```
unless you use `$math mode$` using `$` in the docstring. (The officially recommended way is to use ` `` ` instead of `$` as delimiter for math).

Though I think @HechtiDerLachs recently suggested there were other differences, it would be illuminating to know what else differs